tools/topology/topology2/include/components/tdfb/line2_50mm_azm90_90_13_16khz.conf

```diff
@@ -0,0 +1,361 @@
+# Exported with script example_line_array.m 02-Oct-2023
```

Comment:
Is this the full command line to generate the data ?

Reply:
That's all for Octave or Matlab shell, but I can add second comment line how to build it from SOF tree top. I just added in another PR additional comment print line to these.

Reply:
Thanks, my aim is to make it easy for anyone navigating topology that they can easily work out how to re-tune any module that has coefficients. Long term we need to go a little further and install all the octave/matlab tooling (with a ```sof-``` prefix) so that thay can run 
```
sof-tdfb.m --help
```
and get all the cmd line args or instructions to tune TDFB (and likewise for all modules).